usaspending_api/common/query_with_filters.py

```diff
@@ -379,10 +462,15 @@ class _PlaceOfPerformanceScope(_Filter):
 
     @classmethod
     def generate_elasticsearch_query(cls, filter_value: str, query_type: _QueryType, **options) -> ES_Q:
-        # If an ES record has a pop_country_code of "USA" OR if it has any value for the pop_state_code field
-        #   then it's considered domestic. Since we only support domestic states/territories then any
-        #   pop_state_code value means the pop_country_code would be "USA".
-        pop_scope_query = ES_Q("match", pop_country_code="USA") | ES_Q("exists", field="pop_state_code")
+        if query_type == _QueryType.SUBAWARDS:
+            pop_scope_query = ES_Q("match", sub_place_of_perform_country_co="USA") | ES_Q(
+                "match", sub_place_of_perform_country_name="UNITED STATES"
```

Comment:
`sub_place_of_perform_country_co` and `sub_place_of_perform_country_name` aren't the names of the fields in Elasticsearch, they're `sub_pop_country_code` and `sub_pop_country_name`.

It looks like you used their database column names here.

Reply:
Good catch. Fixed